.github/workflows/labels.yml

```diff
@@ -0,0 +1,23 @@
+name: Check PR Labels
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  check_labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Labels
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const allowedPattern = /^(major|minor|patch)$/;
```

Comment:
I assume we need more allowed labels here-- ie test, internal and/or similar

Reply:
The other labels are "performance", "internal", "documentation", "tests", and "dependencies".